src/refill.rs

```diff
@@ -223,6 +223,14 @@ mod tests {
         assert_eq!(options.line_ending, LineEnding::LF);
     }
 
+    #[test]
+    fn test_unfill_missing() {
+        let (text, options) = unfill("í\n*\n/");
```

Comment:
Could you simplify the test a bit? The `í` could probably be a `foo` and the final `/` could probably be something else, right? Basically, take the fuzzer output and turn it into something a human would write :smile:

Reply:
Hhh, thanks for your comment, I will manually change it for this one and try to add this feature.